plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-additional-information-block/block.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { noticeContexts } from '@woocommerce/base-context';
+import { __ } from '@wordpress/i18n';
+import { StoreNoticesContainer } from '@woocommerce/blocks-components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { CHECKOUT_STORE_KEY } from '@woocommerce/block-data';
+import { ADDITIONAL_FORM_KEYS } from '@woocommerce/block-settings';
+import { Form } from '@woocommerce/base-components/cart-checkout';
+import type { FunctionComponent } from 'react';
+import NoticeBanner from '@woocommerce/base-components/notice-banner';
+
+const Block: FunctionComponent = () => {
+	const { additionalFields, isEditor } = useSelect( ( select ) => {
+		const store = select( CHECKOUT_STORE_KEY );
+		const editorStore = select( 'core/editor' );
+		return {
+			additionalFields: store.getAdditionalFields(),
+			isEditor: !! editorStore,
+		};
+	} );
```

Comment:
Is this how we usually check if we're in the editor? I'm afraid a simple refactor would turn this into an import for the editor. Should be fine for now.

Reply:
I don't think this would be a problem, but I updated it to:

`const { isEditor } = useEditorContext();` which is how it's done in a lot of other places.

We do have mixed uses across the codebase of this way and `useEditorContext`.

Reply:
This is redundant anyway since we're hiding it in the editor too.